### PR TITLE
Added Discord embed limits guide

### PR DESCRIPTION
**Pull request by kkrypt0nn**
This adds the Discord embed limits pinned message as a guide, as asked by @Shom770.

### CODE REVIEW

**Review by Shom770 — APPROVED** [review]
LGTM!

File: pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- Embed **footer** is limited to **2048 characters**
+- Embed **author name** is limited to **256 characters**
+- The **total of characters** allowed in an embed is **6000**
+
+Now if you need to get over this limit (for example for a help command), you would need to use pagination.
+There are several ways to do that:
+
+- A library called `disputils` -> <https://pypi.org/project/disputils>
+- An experimental library made by the discord.py developer called `discord-ext-menus` -> <https://github.com/Rapptz/discord-ext-menus>
+- Make your own setup using the `wait_for()` and wait for a reaction to be added -> <https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.wait_for>
```
Comment (HassanAbouelela) [inline pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md]: This URL breaks the page on mobile (see screenshot). Not sure why it's only this link, and I don't know if this is related to anything you did in particular, but I haven't had a chance to investigate.

image

**Review by HassanAbouelela — CHANGES_REQUESTED** [review]
Blocking temporarily, see my comment.

File: pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md (hunk 1 above)
Comment (kkrypt0nn) [reply pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md]: I could use a hyperlink instead.

Comment (sue-m-oh) [reply pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md]: I think if you use hyperlinks for these 3 bullet points we should be good to go @kkrypt0nn 

I would suggest using bolded text, and not inline code block for these links. Example: 

image

Comment (kkrypt0nn) [reply pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md]: Sure thing, will be done next week 👍

**Comment by Xithrius** [comment]
I don't know why I removed all the tags before, lol

They're fixed now.

**Review by wookie184 — APPROVED** [review]
LGTM

**Review by Xithrius — APPROVED** [review]
I approve of this message.

**Pull request merged**